sp/src/game/server/ez2/npc_wilson.cpp

```diff
@@ -128,7 +128,10 @@ BEGIN_DATADESC(CNPC_Wilson)
 
 	DEFINE_KEYFIELD( m_bDead, FIELD_BOOLEAN, "dead" ),
 
-	DEFINE_INPUT( m_bOmniscient, FIELD_BOOLEAN, "SetOmniscient" ),
+	DEFINE_INPUT( m_bOmniscient, FIELD_BOOLEAN, "SetOmniscient" ), // TODO: Replace with "Omnipresent" when saves can be changed
```

Comment:
Are you positive this has never been used before? It seems kind of dangerous to change the name of an input now. I don't recall if we used it in Chapter 3, but it might be used in some maps. I'd leave the name alone if we can.